janus.h

```diff
@@ -171,6 +177,14 @@ int janus_process_success(janus_request *request, json_t *payload);
  * @returns 0 on success, a negative integer otherwise
  */
 int janus_process_error(janus_request *request, uint64_t session_id, const char *transaction, gint error, const char *format, ...) G_GNUC_PRINTF(5, 6);
+/*! \brief Method that handles session set_timeout request
+ * @param[in] request The request instance and its source
+ * @param[in] session The Janus session
+ * @param[in] transaction The Janus transaction identifier
+ * @param[in] root The request payload root
+ * @returns 0 on success, a negative integer otherwise
+ */
+int janus_process_set_session_timeout(janus_request *request, janus_session *session, const gchar *transaction, json_t *root);
```

Comment:
See above. I actually think that, besides removing them from the header, we can get rid of both methods and do it more simply inline where the requests are processed, since the update functionality is simple.